ReactSkia/views/common/RSkSdkConversion.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 1994-2022 OpenTV, Inc. and Nagravision S.A.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+
```

Comment:
namespace rns {
namespace sdk {
}
}

Reply:
Currently it is placed under Path "[ReactSkia/views/common]".
Is it ok  to have file parts of rns:sdk namespace in different folder other than ReactSkia/sdk. or I have to move the RSkSdkConversion.h file in to sdk ?

Reply:
As discussed, retained the folder Location , considering conversion functionality from RSK to sdk is provided by ReactSkia instead of sdk and added enclosed conversion under namespace facebook:react